### PR TITLE
update literal-values.md

### PR DESCRIPTION
**Pull request by tangenta**
### What is changed, added or deleted? (Required)
- Add summary and correct some description in literal-values.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: literal-values.md [hunk 1]
```diff
@@ -25,11 +28,13 @@ String Literals 是一个 bytes 或者 characters 的序列，两端被单引号
 
 如果 `ANSI_QUOTES` SQL MODE 开启了，那么只有单引号内的会被认为是 String Literals，对于双引号内的字符串，会被认为是一个 identifier。
 
+字符串分为两种：二进制字符串(binary string)和非二进制字符串。二进制字符串由字节序列构成，它的 charset 和 collation 都是 `binary`，在互相比较时利用**字节**作为单位；非二进制字符串由字符序列构成，有除 `binary` 以外的多种 charset 和 collation，在相互比较时用**字符**（一个字符可能包含多个字节，取决于 charset 的选择）作为单位。
```
Comment (TomShawn) [inline literal-values.md:31]:
```suggestion
字符串分为以下两种：

+ 二进制字符串 (binary string)：由字节序列构成，它的 charset 和 collation 都是 `binary`，在互相比较时利用**字节**作为单位。
+ 非二进制字符串：由字符序列构成，有除 `binary` 以外的多种 charset 和 collation，在相互比较时用**字符**（一个字符可能包含多个字节，取决于 charset 的选择）作为单位。
```

File: literal-values.md [hunk 2]
```diff
@@ -25,11 +28,13 @@ String Literals 是一个 bytes 或者 characters 的序列，两端被单引号
...
+
 binary string 是一串 bytes 组成的字符串，每一个 binary string 有一个叫做 `binary` 的 character set 和 collation。一个非二进制的字符串是一个由字符组成的字符串，它有除 `binary` 外的 character set和与之兼容的 collation。
```
Comment (TomShawn) [inline literal-values.md:33]: 这一段和上面内容重复。

File: literal-values.md [hunk 3]
```diff
@@ -55,26 +60,23 @@ SELECT n'some text';
...
-- \\b: 退格符号
-- \\n: 换行符
-- \\r: 回车符
-- \\t: tab 符（制表符）
-- \\z: ASCII 26 (Ctrl + Z)
-- \\\\: 反斜杠 \\
-- \\%: \%
-- \\_: \_
+| 转义字符 | 含义 |
+| ------ | ---- |
```
Comment (TomShawn) [inline literal-values.md:66]:
```suggestion
| :------ | :---- |
```

**Comment by TomShawn** [comment]
@kissmydb PTAL

File: literal-values.md [hunk 4]
```diff
@@ -1,9 +1,12 @@
 ---
 title: 字面值
 category: reference
+summary: 本文介绍了 TiDB SQL 语句的字面值。
 aliases: ['/docs-cn/dev/reference/sql/language-structure/literal-values/']
 ---
 
+TiDB 字面值包括字符字面值、数值字面值、时间日期字面值、十六进制、二进制字面值和 NULL 字面值。以下分别对这些字面值进行一一介绍。
```
Comment (ran-huang) [inline literal-values.md:8]: This paragraph should be put under the title.

Comment (TomShawn) [reply literal-values.md:8]: Fixed in https://github.com/pingcap/docs-cn/pull/3845. PTAL

**Pull request merged**